packages/editor/src/store/actions.js

```diff
@@ -769,33 +769,123 @@ const getBlockEditorAction = ( name ) => function* ( ...args ) {
 	yield dispatch( 'core/block-editor', name, ...args );
 };
 
+/**
+ * @see resetBlocks in core/block-editor store.
+ */
 export const resetBlocks = getBlockEditorAction( 'resetBlocks' );
+/**
```

Comment:
These all look great but from a code standards/ bike sheddy perspective I just wonder if we should have newlines above all these, similar to how it'd be done in PHP doc?

Reply:
let's do that d1b7afffa98f0170ba332df2f7f0a46762a6e20c